docs/install.md

```diff
@@ -22,9 +22,20 @@ make helm-upgrade-keda
 
 >This document will rely on environment variables such as `${NAMESPACE}` to indicate a value you should customize and provide to the relevant command. In the above `helm install` command, `${NAMESPACE}` should be the namespace you'd like to install KEDA into. KEDA must be installed in every namespace that you'd like to host KEDA-HTTP powered apps.
 
+## Install via Helm Chart
+
+This repository is within KEDA's default helm repository on [kedacore/charts](http://github.com/kedacore/charts), you can install it by running:
+
+```console
+helm repo add kedacore https://kedacore.github.io/charts
+helm repo update
+
+helm install http-add-on kedacore/keda-add-ons-http --create-namespace --namespace keda
+```
+
 ## Installing HTTP Components
 
-This repository comes with a Helm chart built in. To install the HTTP add on with sensible defaults, first check out this repository and `cd` into the root directory (if you haven't already):
+This repository also comes with a Helm chart built in. To install the HTTP add on with sensible defaults, first check out this repository and `cd` into the root directory (if you haven't already):
```

Comment:
@khaosdoctor let's follow keda's lead and not ship this repository with the chart built into it. this will keep kedacore/charts the canonical and exclusive source for the http-add-on charts, and we can remove this entire section.

Reply:
Moved, I think you'll need to close #150 to avoid conflicts, I already moved your changes here on 2672e17

Reply:
@khaosdoctor great, thank you! I closed #150 just now.